fli-iam/shanoir-ng
# shanoir-issue#2375: special char in ohif study list

How to test:
Open the ohif viewer from solr
Click on the arrow to go to the study list in ohif:
![image](https://github.com/user-attachments/assets/4ea1e65e-54c5-42c7-bdaa-87618a454459)
Study names should be displayed correctly
